index.html

```diff
@@ -51,111 +55,14 @@ <h1>I’m Alqama <br> Glad to see you!</h1>
-                </div>
-                
-                <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="list">
-                    <li class="box"><button type="button">html</button></li>
-                    <li class="box"><button type="button">css</button></li>
-                    <li class="box"><button type="button">javaScript</button></li>
-                </ul>
-                
-                <button class="card_see" type="button"> See Project</button>
-            </div>
-        </div>
-        
-        <div class="card">
-            <div class="card_pic">
-                <img src="./images/SnapshootPortfolio(2).png" alt="GymFit image" class="snapshoot">
-            </div>
-            <div class="transform">  
-                <h1>Tonic</h1>
-                
-                <div class="sec">
-                <p>CANOPY</p>
-                <span class="dot"></span>
-                <p class="sec_color1">Back End Dev</p>
-                <span class="dot"></span>
-                <p class="sec_color2">2015</p>
-            </div>
-            
-            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="list">
-                <li class="box"><button type="button">html</button></li>
-                <li class="box"><button type="button">css</button></li>
-                <li class="box"><button type="button">javaScript</button></li>
-            </ul>
-            
-            <button class="card_see" type="button">See Project</button>
-            </div>
-        </div>
+<div class="space" id="works"></div>
 
-        <div class="card tra_desk">
-            <div class="card_pic">
-                <img src="./images/SnapshootPortfolio(3).png" alt="Art Printing" class="snapshoot snap">
-            </div>
-            <div class="transform">
-            <h1>Multi-Post Stories</h1>
-             
-             <div class="sec">
-                <p>CANOPY</p>
-                <span class="dot"></span>
-                <p class="sec_color1">Back End Dev</p>
-                <span class="dot"></span>
-                <p class="sec_color2">2015</p>
-            </div>
-            
-            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="list">
-                <li class="box"><button type="button">html</button></li>
-                <li class="box"><button type="button">css</button></li>
-                <li class="box"><button type="button">javaScript</button></li>
-            </ul>
-            
-            <button class="card_see" type="button">See Project</button>
-            </div>
-        </div>
-    </section>
+<section class="works make-flex">
+
+</section>
 
     <section class="card aboutme" id="about-myself">
         <div class="aboutme_desk">
```

Comment:
- [ ] `optional`
- [ ] Good implimenting some of the changes
- [ ] Kindly move the `frame & skills` on the bottom the languages
- [ ] please see screenshot

`your figma template`
![image](https://user-images.githubusercontent.com/66230306/189332076-af04be84-c9b1-409a-a963-254598fe4d27.png)

`figma template`
![image](https://user-images.githubusercontent.com/66230306/189332818-5eebaa03-fb23-4f21-9e30-766555619610.png)